### PR TITLE
template: add zoom link for user-feedback

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@dshaw can you review?

**Comment by mhdawson** [comment]
@dshaw ping !

**Comment by mhdawson** [comment]
@dshaw ping

File: templates/meeting_base_userfeedback [hunk 1]
```diff
@@ -6,5 +6,6 @@ GROUP_NAME="User Feedback"
 AGENDA_TAG=user-feedback-agenda
 JOINING_INSTRUCTIONS="
 
-* link for participants: Will be added a few minutes before meeting starts
-* For those who just want to watch: https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ/live"
+* link for participants: https://zoom.us/j/353642832
+* For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
+* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled
```
Comment (thefourtheye) [inline templates/meeting_base_userfeedback]: Shouldn't this end with `"`?

**Comment by mhdawson** [comment]
@thefourtheye thanks for the catch, updated to reflect your comment.

File: templates/meeting_base_userfeedback [hunk 2]
```diff
@@ -6,5 +6,6 @@ GROUP_NAME="User Feedback"
 AGENDA_TAG=user-feedback-agenda
 JOINING_INSTRUCTIONS="
 
-* link for participants: Will be added a few minutes before meeting starts
-* For those who just want to watch: https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ/live"
+* link for participants: https://zoom.us/j/353642832
+* For those who just want to watch We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.
```
Comment (thefourtheye) [inline templates/meeting_base_userfeedback]: Sorry I didn't catch this earlier. "For those who just want to watch" should be removed. Also, a sentence break after "listen to it live" would be better.

**Comment by mhdawson** [comment]
@thefourtheye addressed your comments.

**Comment by dshaw** [comment]
@mhdawson My apologies. My GitHub notifications sometimes get buried. 👀

**Pull request merged**